frontend/src/scenes/insights/views/InsightsTable/InsightsTable.scss

```diff
@@ -3,7 +3,6 @@
     align-items: center;
 
     .edit-icon {
-        font-size: 1rem;
```

Comment:
Other than this affecting the snapshots it all looks good (mostly trusting the snapshot system to have caught any issues)

Reply:
Yeah, worked through a lot of issues thanks to the snapshots. Fixed the icon size up so there's only (one) intentional change now

Reply:
Going to go ahead and merge this now that there is only a single snapshot change